repos/spack_repo/builtin/packages/meep/package.py

```diff
@@ -83,7 +84,9 @@ class Meep(AutotoolsPackage):
     def configure_args(self):
         spec = self.spec
 
-        config_args = ["--enable-shared"]
+        config_args = ["LDFLAGS=-L{0}".format(spec["fftw"].prefix.lib)]
```

Comment:
```suggestion
        config_args = ["LDFLAGS=-L{0}".format(spec["fftw-api"].prefix.lib)]
```
:slightly_smiling_face:

Reply:
Also, it'd be better to ask via `.libs` instead of hard-coding `prefix.lib`

Reply:
I haven't used .libs so far. Is this covered somewhere in https://spack.readthedocs.io/en/latest/packaging_guide_build.html ? Or do you have an example of another spack package that uses .libs such that I can have a look at how this works?